README.md

```diff
@@ -55,6 +55,49 @@ color output from `DefaultReporter`:
 Minitest::Reporters.use! [Minitest::Reporters::DefaultReporter.new(:color => true)]
 ```
 
+
+## SpecReporter - configuration options ##
+
+The SpecReporter supports the customised output as highlighted below.
+
+By default the output would look something like this:
+
+```ruby
+"test_0001_should return the correct path                        PASS (0.00s)"
+```
+
+
+But the output can be changed via these configuration options:
+
+ *  `:show_order`   - sets the output order (`:before` or `:after`) [ Default: `:after` ]
```

Comment:
imho it is unclear what order is controlled here :(
You need either improve the documentation or change option's name.